test-suite/coq-makefile/findlib-package/run.sh

```diff
@@ -7,7 +7,8 @@ export OCAMLPATH=$OCAMLPATH:$PWD/findlib
 if which cygpath 2>/dev/null; then
   # the only way I found to pass OCAMLPATH on win is to have it contain
   # only one entry
-  export OCAMLPATH=`cygpath -w $PWD/findlib`
+  OCAMLPATH=$(cygpath -w "$PWD"/findlib)
```

Comment:
As above, technicall fine, but as above the export split is not consistent (elsewhere export var=value is used)

Reply:
As it was said above, we must separate assigning and exporting when the value comes from a command, so that, the script will fail if the command fails.